chainer/variable.py

```diff
@@ -68,6 +68,46 @@ def __repr__(self):
     def __str__(self):
         return self.name or ('<var@%x>' % id(self))
 
+    def debug_print(self):
+        """Display stored data and location of the Variable"""
```

Comment:
I think "Display **summary** of stored data" is more appropriate because it does not show whole data.